.appveyor.yml

```diff
@@ -45,7 +45,7 @@ install:
     - echo extension=php_curl.dll >> php.ini-max
     - copy /Y php.ini-max php.ini
     - cd c:\projects\symfony
-    - IF NOT EXIST composer.phar (appveyor DownloadFile https://github.com/composer/composer/releases/download/2.0.0/composer.phar)
+    - IF NOT EXIST composer.phar (appveyor DownloadFile https://github.com/composer/composer/releases/download/2.0.13/composer.phar)
```

Comment:
maybe we should download the latest composer 2 phar from https://getcomposer.org/download/latest-2.x/composer.phar instead (as done in github actions and Travis, which always use an uptodate composer version) ?